ghost/core/core/server/web/api/endpoints/admin/middleware.js

```
@@ -14,16 +15,34 @@ const messages = {
  * @param {import('express').NextFunction} next
  */
 const notImplemented = function notImplemented(req, res, next) {
-    // CASE: user is logged in, allow
+    // CASE: user is logged in with user auth, skip to permission system
     if (!req.api_key) {
         return next();
     }
 
+    // CASE: user is requesting with staff token, check blocklist, else skip to permission system
+    // Staff tokens have a user_id associated with them, integration tokens don't
+    if (req.api_key && req.api_key.get('user_id')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)